playbooks/common/openshift-cluster/upgrades/etcd/rhel_tasks.yml

```diff
@@ -3,12 +3,7 @@
   command: "etcdctl --cert-file /etc/etcd/peer.crt --key-file /etc/etcd/peer.key --ca-file /etc/etcd/ca.crt -C https://{{ openshift.common.hostname }}:2379 cluster-health"
 
 - name: Update etcd package but exclude etcd3
```

Comment:
Maybe update the name since exclude is no long used.  Also, possibly switch to `package` module since the ``--exclude` flag is no longer necessary.